### ISSUE TITLE
Error applying Draco compression to model

### ISSUE THREAD

**Issue by looeee** [donmccurdy/glTF-Transform#259]
Another error from one of the poly models. Let me know if I should report this to draco3dgltf instead of here. 

Running `gltf-transform draco .\model.gltf .\model_d.glb -v` returns this error:

``` 
debug: weld: Complete.
debug: [KHR_draco_mesh_compression] Compression options: {"method":1,"encodeSpeed":5,"decodeSpeed":5,"quantizationBits":{"POSITION":14,"NORMAL":10,"COLOR":8,"TEX_COORD":12,"GENERIC":12}}
```

Possibly caused by the "GENERIC" attribute? 

Model: [3D scanned skull_aavt.zip](https://github.com/donmccurdy/glTF-Transform/files/6525872/3D.scanned.skull_aavt.zip)

**Comment by donmccurdy** [comment]
Hm, I'm not sure what's going on with this error. It seems to depend on the encoding method — if you add `--method sequential` to the parameters it will succeed, although that does reduce the compression rate quite a bit... it's there if you need compression that preserves vertex order. 

I'm guessing this might be a bug in the Draco encoder, but I'll try to figure out the exact inputs to the Draco encoder here since the Draco team might need that to debug it.

**Comment by donmccurdy** [comment]
Or equivalently:

```ts
import { DracoMeshCompression } from '@gltf-transform/extensions';

document.createExtension(DracoMeshCompression)
    .setRequired(true)
    .setEncoderOptions({
        method: DracoMeshCompression.EncoderMethod.EDGEBREAKER
    });
```

**Comment by looeee** [comment]
OK, thanks for checking it out. There are several models with both this and the previous error so for now I'm just moving the unprocessed models to a separate folder. 

Here's another one with this error. Also has "scanned" in the name. This one says it was scanned with www.qlone.pro. The previous one doesn't have any data about what app was used for scanning. 

[A seashell scanned with Qlone_1ric.zip](https://github.com/donmccurdy/glTF-Transform/files/6525914/A.seashell.scanned.with.Qlone_1ric.zip)

**Comment by looeee** [comment]
And one more. 
Presumably this one wasn't scanned. 

[Barack Obama_czi3.zip](https://github.com/donmccurdy/glTF-Transform/files/6525922/Barack.Obama_czi3.zip)

**Comment by donmccurdy** [comment]
Hm both models have a consistent issue — a buffer view with its `byteStride` property set to 0, which is completely invalid (the minimum is 4: https://github.com/KhronosGroup/glTF/blob/master/specification/2.0/README.md#bufferviewbytestride) but I guess three.js just happens to treat that as if the bytestride weren't there at all, and use the default.

Definitely an issue in the model's exporter, but might be easy to patch if you can modify the JSON before loading it...

```ts
const jsonDocument = io.readAsJSON('path/to/model.gltf');

jsonDocument.json.bufferViews.forEach((bufferView) => {
  if (bufferView.byteStride === 0) {
    delete bufferView.byteStride;
  }
});

const document = io.readJSON(jsonDocument);
// ... everything else should work now
```

**Comment by donmccurdy** [comment]
See https://github.com/donmccurdy/glTF-Transform/issues/260#issuecomment-846460680 – i think this is fixed and the example I'm testing works in v0.11.1, requiring that fix to the invalid buffer views. But let me know if Draco still isn't working on your side.
